kernel-crawler/kernel-crawler.py

```diff
@@ -287,8 +287,9 @@
             "type": "s3",
             "root": "https://storage.googleapis.com/cos-tools",
             "patterns": [
-                "\d+\.\d+\.\d+/kernel-src.tar.gz$",
-                "\d+\.\d+\.\d+/kernel-headers\.t(ar\.)?gz$",
+                # Only include kernels that are part of LTS releases https://cloud.google.com/container-optimized-os/docs/release-notes
+                "(16108|16623|16919|17162|17412)\.\d+\.\d+/kernel-src.tar.gz$",
+                "(16108|16623|16919|17162|17412)\.\d+\.\d+/kernel-headers\.t(ar\.)?gz$",
```

Comment:
Will we have to update this manually when a new version of COS rolls around? I'm not super familiar with their versioning but those numbers look like they correspond to some kind of milestone, so I'd assume the next LTS release will have a different version number.

Alternative take is that we'll only support COS via CO-RE BPF for future LTS versions, on comment clarifying that would be nice if that's the case.

Reply:
Good point. Yes, these are LTS release numbers and we'd have to update this for future versions. I think we just need a better filter. I'll remove this for now.